sstable-core/src/main/java/com/fullcontact/sstable/hadoop/mapreduce/SSTableRecordReader.java

```diff
@@ -42,9 +42,9 @@
  * @author ben <ben.vanberg@fullcontact.com>
  */
 public abstract class SSTableRecordReader<K, V> extends RecordReader<K, V> {
-
+    public static final String SSTABLE_INPUT_FILES_ISCOMPRESSION = "hadoop-sstable.input.files.isCompression";
```

Comment:
Could you move this to https://github.com/fullcontact/hadoop-sstable/blob/master/sstable-core/src/main/java/com/fullcontact/sstable/hadoop/mapreduce/HadoopSSTableConstants.java and follow the same pattern established there for constant naming?